doc/en/example/simple.rst

```diff
@@ -895,29 +895,35 @@ here is a little example implemented via a local plugin:
 
     import pytest
 
+    your_unique_key = StashKey[Dict[str, bool]]()
```

Comment:
`your_unique_key` is not a very good name. How about `phase_report_key`, if you choose to store the full report, or `phase_result_key` if not?

Reply:
e20dfb6

Yeah but isn't `stash` shared with all the other plugins and stuff? If that's the case, it should be remarked that the key name should be unique to avoid collision.

Reply:
> Yeah but isn't stash shared with all the other plugins and stuff?

The name doesn't matter, every stash and stash key (i.e. `StashKey()`) pair are unique.

Reply:
Understood. I was confused about it.